sdk/src/main/java/com/bugsnag/android/Error.java

```diff
@@ -112,7 +134,7 @@ public void setContext(String context) {
      */
     @Nullable
     public String getContext() {
-        if (!TextUtils.isEmpty(context)) {
+        if (context != null) {
```

Comment:
what's the intent of this change?

Reply:
This was unintentional - I've reverted back to the TextUtils.isEmpty approach which is more robust.